k8s/prod/common/rd/judicial-api.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: rd-judicial-api
+  namespace: rd
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: rd-judicial-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rd-judicial-api
+    version: 0.0.4
```

Comment:
your base chart  version needs updating before going to prod:
https://github.com/hmcts/rd-judicial-api/blob/master/charts/rd-judicial-api/requirements.yaml#L3

see here for the latest: https://github.com/hmcts/chart-java/releases

Reply:
okay..will do it now and release the chart version update first1 👍